solr/core/src/java/org/apache/solr/core/SolrResourceLoader.java

```diff
@@ -529,6 +536,18 @@ public String resourceLocation(String resource) {
     
     Class<? extends T> clazz = null;
     try {
+      // If there is a package name prefix ...
+      Pair<String, String> pkgClassPair = PluginInfo.parseClassName(cname);
+      PackageLoader.Package pkg = getPackage(pkgClassPair.first());
+      if (pkg == null) {
+        // essentially, remove the package prefix and continue as normal.  Maybe it'll be found.
+        cname = pkgClassPair.second();
+      } else {
+        // TODO what version?
```

Comment:
Trying to load the latest always? Why? Need to revisit.

Reply:
Yes, I have a "TODO" at this line.  How do I get the right version?